src/gui/gui.pri

```diff
@@ -79,6 +78,28 @@ win32|macx {
     SOURCES += $$PWD/programupdater.cpp
 }
 
+# notifications
+HEADERS += \
+    $$PWD/notifications/notifier.h
+
+SOURCES += \
+    $$PWD/notifications/notifier.cpp
+
+unix:!macx:dbus {
```

Comment:
Well. In this case, the following would be correct:
```qmake
unix:!macx:dbus {
    include(notifications/qtnotify/qtnotify.pri)
    HEADERS += $$PWD/notifications/dbusnotifier.h
    SOURCES += $$PWD/notifications/dbusnotifier.cpp
} else {
    HEADERS += $$PWD/notifications/systemtraynotifier.h
    SOURCES += $$PWD/notifications/systemtraynotifier.cpp
}
```
Also note that there is no need for extra line breaks.